source/Cosmos.System2/FileSystem/Disk.cs

```diff
@@ -280,6 +289,8 @@ public void MountPartition(int index)
 
                     //We would have done Partitions[i].MountedFS = item.Create(...), but since the array is not cached, we need to store the mounted partitions in a list
                     MountedPartitions[index] = item.Create(part.Host, xRootPath, xSize);
+                    // the array is now chased
+                    //part.MountedFS = item.Create(part.Host, xRootPath, xSize);
```

Comment:
If code is no longer needed please remove it rather than commenting it out

Reply:
I removed it